packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -2034,6 +2034,7 @@ export class Task {
     private taskSource: string;
     private taskGroup: TaskGroup | undefined;
     private taskPresentationOptions: theia.TaskPresentationOptions;
+    private _runOptions: theia.RunOptions;
```

Comment:
While would prefer that we keep the naming of the fields consistent inside the same class: let's fix all of the names to follow the `_` convention or call the field `taskRunOptions`.